CITATION.cff

```diff
@@ -2,6 +2,6 @@ message: "If you use this software, please cite it as below."
 authors:
 - family-names: "Cook County Assessor's Office"
 title: "AssessPy"
-version: 1.2.0
-date-released: 2022-11-14
+version: 2.0.0
```

Comment:
Need a major version bump due to the incompatible API changes. I'll also update `assessr` and try to keep the two in parity.